JavaScript/CodingConventions.md

```diff
@@ -477,11 +499,37 @@ function makeCookies(flavor) {
 
 We use JSDoc style comments above function declarations and at the top of files because they make code clear and easy to read, and we'd like to be able to generate JavaScript documentation at some point.
 
+The description for parameters is optional; description for `@returns` statement is not optional. Also it's encouraged to have description and params separated by newline.
+
+**Warning**: Use of inline comments in not allowed:
+```javascript
+// bad
+var a = 1; // declaring a to 1
+
+function getRandomNumber(){
+    return 4; // chosen by fair dice roll.
+              // guaranteed to be random.
+}
+
+/* A block comment before code */ var b = 2;
+
+var c = 3; /* A block comment after code */
+```
+
+
+#### Single-line comments
+All single line comments should begin with space. Consider this:
+```javascript
+/////// this is  bad comment
```

Comment:
At first look this seems confusing (there is a space before "this"!). IMO `//this is bad comment` would be clearer example.